web/profiles/webspark/webspark/webspark.install

```diff
@@ -277,3 +277,24 @@ function webspark_update_9012(&$sandbox) {
   // Lock the configuration storage.
   \Drupal::state()->set('configuration_locked', TRUE);
 }
+
+/**
+ * Install imagemagick module and dependencies.
+ */
+function webspark_update_9013(&$sandbox) {
+  $module_installer = \Drupal::service("module_installer");
+
+  $module_installer->install(["file_mdm"]);
+  $module_installer->install(["sophron"]);
+  $module_installer->install(["imagemagick"]);
```

Comment:
Hi @mlsamuelson 
I realized this module is not being installed despite of this line : 
https://pr-2-webspark-ci.ws.asu.edu/admin/modules

Reply:
However, locally it works.

Reply:
Hi @mlsamuelson 
After investigate a little bit more about these modules not being installed. I found out that we installed this first, however they quickly uninstalled around the same time. Please check the attached screenshot where I display the Drupal logs report section from this box https://pr-2-webspark-ci.ws.asu.edu : 
![image](https://user-images.githubusercontent.com/33984278/236505232-54ea72f8-cf64-4e36-8371-39f70c426d6e.png)

Also, I found into this config file [core.extension.yml ](https://github.com/ASUWebPlatforms/webspark-ci/blob/master/config/core.extension.yml) that we don't have those modules, is it possible that I must also add these modules into this core extension file?

- https://github.com/ASUWebPlatforms/webspark-ci/blob/master/config/core.extension.yml

CC @bebaps , @tbutterf